gallery/visualtests/nxCombobox.js

```diff
@@ -0,0 +1,210 @@
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { setupBrowser } = require('./testUtils');
+
+describe('NxCombobox', function() {
+  const {
+    waitAndGetElements,
+    getPage,
+    simpleTest,
+    blurElement,
+    disableLoadingSpinnerAnimation,
+    checkScreenshot,
+    a11yTest
+  } = setupBrowser('#/pages/Combobox');
+
+  const basicExampleSelector = '#nx-combobox-basic-example .nx-combobox',
+      nonEmptinessExampleSelector = '#nx-combobox-non-emptiness-example .nx-combobox',
+      backendExampleSelector = '#nx-combobox-backend-example .nx-combobox',
+      errorExampleSelector = '#nx-combobox-error-example .nx-combobox',
+      disabledExampleSelector = '#nx-combobox-disabled-example .nx-combobox';
+
+  describe('when loading', function() {
+    beforeEach(async function() {
+      const inputSelector = `${backendExampleSelector} .nx-combobox__input input`,
+          loadingSpinnerSelector = `${backendExampleSelector} .nx-loading-spinner`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('1');
+      await getPage().waitForSelector(loadingSpinnerSelector);
+    });
+
+    it('looks right', async function() {
+      const [component] = await waitAndGetElements(backendExampleSelector);
+      await disableLoadingSpinnerAnimation();
+      await checkScreenshot(component, 300, 130);
+    });
+
+    it('passes a11Y checks', a11yTest());
+  });
+
+  describe('when displaying results', function() {
+    beforeEach(async function() {
+      const inputSelector = `${backendExampleSelector} .nx-combobox__input input`,
+          dropdownButtonSelector = `${backendExampleSelector} .nx-dropdown-button`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('1');
+      await getPage().waitForSelector(dropdownButtonSelector);
+    });
+
+    it('looks right', async function() {
+      const [component] = await waitAndGetElements(backendExampleSelector);
+      await checkScreenshot(component, 300, 376);
+    });
+
+    it('passes a11Y checks', a11yTest());
+  });
+
+  describe('with long variant', function() {
+    beforeEach(async function() {
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          dropdownButtonSelector = `${basicExampleSelector} .nx-dropdown-button`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('a');
+      await getPage().waitForSelector(dropdownButtonSelector);
+    });
+
+    it('looks right', async function() {
+      const [component] = await waitAndGetElements(basicExampleSelector);
+      await checkScreenshot(component, 800, 376);
+    });
+
+    it('passes a11Y checks', a11yTest());
```

Comment:
I don't think the long and short variants need their own a11y checks.

Reply:
Updated here a59e762c3a1d648b3d67c8e5a9ec4e7ff9ec06ce.